### PR TITLE
Update post-process: 1st step - worked on creating refined mesh with the correct ID markers and ordered nodal coordinate

### PR DESCRIPTION
**Pull request by keiyamamo**
This is work-in-progress and just an example of how we can modify post-process scripts. 
We can look through this together during the meeting. @dbruneau-mie

### CODE REVIEW

**Comment by keiyamamo** [comment]
There are many other things that I need to work on for the post-processing, but this PR itself is ready for review.
Here is a summary of the PR. 

1. I put license identifier at the top of the each file following the syntax of [dolfinx](https://github.com/FEniCS/dolfinx/blob/270438d0bfd165aade0875502c61ea26748083d8/python/dolfinx/__init__.py#L5) Right now, it is not decided how we put license in each files so I thought it would be nice to bring this up. I can make changes to the existing files if we agree on how to put license. 

2. I also put Copyright as you can see in `create_refined_mesh.py`. Copyright is also something we didn’t fully decide what to do, but I put @dbruneau-mie ’s name on this file since it is completely clear that it’s his original work. I don’t know if I should put Copyright in `__init__.py` since there’s very little code in it. If there’s any insight, it would be much appreciated!

3. I modified the original version of `create_refined_mesh.py`. The main purpose of this script is still the same. My work was to be more explicit about what the script is executing and also inserted some checks to reduce errors. Generally, I tried to follow the syntax from @johannesring ’s meshing script. 

This does not have to be merged right now, but would be nice to discuss some details with @dbruneau-mie in the next meeting, and hope to have any comments for improvement from @johannesring  and @jorgensd .

File: src/fsipy/automatedPostprocessing/postprocessing_mesh/create_refined_mesh.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2023 David Bruneau
+# Modified by Kei Yamamoto 2023
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+"""
+This script creates a refined mesh with domain markers from a specified mesh. However, the node numbering may not be the
+same as the output file with save_deg = 2, so the node numbering is corrected to match the output file
```
Comment (johannesring) [inline src/fsipy/automatedPostprocessing/postprocessing_mesh/create_refined_mesh.py:7]: Remove "output file" at the end of this line.

File: src/fsipy/automatedPostprocessing/postprocessing_mesh/create_refined_mesh.py [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        indexMap = np.append(orederedWrongNodes, orederedCorrectNodes, axis=1)
+        orderedIndexMap = indexMap[indexMap[:, 0].argsort()]
+        orderedIndexMap = orderedIndexMap[:, [0, 4]]
+
+        # wrongNumberTopology is the topology from the "wrong" numbered mesh.
+        # We will modify this array to change it to the correct node numbering scheme
+        wrongNumberTopology = wrongNumberMesh['mesh/topology'][:]
+
+        # This loop replaces the node numbers in the topology array one by one
+        print('--- Correcting node numbering of the tpology array in the refined mesh \n')
```
Comment (johannesring) [inline src/fsipy/automatedPostprocessing/postprocessing_mesh/create_refined_mesh.py:118]: Typo her (_tpology_).

**Review by johannesring — APPROVED** [review]
Looks good!

**Comment by keiyamamo** [comment]
@johannesring  Thank you for the review!

**Pull request merged**